packages/bgpdisco/bgpdisco_nameservice.conf

```diff
@@ -0,0 +1,11 @@
+package 'bgpdisco-plugin-nameservice'
+
+config general
+	option domain 'ff'
```

Comment:
I propose to use `.internal` here as it is now a reserved domain for internal use or a proper domain as suffix, as `.ff` might get registered in the future.

Reply:
There is no country code for FF. Furthermore two letter TLDs are reserved for countries. Unless noone wants to assign FF it's free. Tbh, i perefer `ff` over a longer `internal` or `freifunk` simply for convinience

https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Decoding_table

Reply:
From what I know the reservation of 2 character domains is gone for quite some time... (https://www.icann.org/en/blogs/details/icann-clears-the-way-for-two-character-second-level-domain-names-12-11-2014-en)